.github/DEVELOPMENT.md

```diff
@@ -10,6 +10,10 @@ yarn
 
 > Tip: Consider using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) and running `nvm install` to sync the Node version used to develop on this repository.
 
+## Linting
+
+To lint the codebase, run `yarn run webext:lint`. This uses the [addons linter](https://github.com/mozilla/addons-linter) to check for common errors. For available options, see the [addons linter documentation](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#web-ext-lint).
```

Comment:
[Bug] `lint:spelling` is failing with new words like `webext`. You can plop them into `cspell.json` as known words.